src/main/java/org/broadinstitute/hellbender/cmdline/GATKPlugin/GATKReadFilterPluginDescriptor.java

```diff
@@ -204,6 +204,18 @@ public boolean isDependentArgumentAllowed(final Class<?> dependentClass) {
      */
     @Override
     public void validateArguments() {
+        // throw if any filter is specified twice
+        final List<String> moreThanOnce = userReadFilterNames.stream()
+                .collect(Collectors.groupingBy(e -> e, Collectors.counting()))
+                .entrySet().stream().filter(e -> e.getValue() != 1)
+                .map(e -> e.getKey() + " (" + e.getValue() + ")")
+                .collect(Collectors.toList());
```

Comment:
I think the counts are overkill; this could be much simpler:

```
final Set<String> uniqueUserFiltersNames = new HashSet<>();
final Set<String> duplicateUserFilterNames = userReadFilterNames.stream().filter(name -> !uniqueUserFiltersNames.add(name)).collect(Collectors.toSet());
```